implementations/rust/ockam/ockam_api/src/config/lookup.rs

```diff
@@ -11,9 +11,11 @@ use std::{
 #[derive(Debug, Default)]
 pub struct LookupMeta {
     /// Append any project name that is encountered during look-up
-    pub project: VecDeque<String>,
+    pub project: VecDeque<Name>,
```

Comment:
`LookupMeta::project` is never read. What is the purpose of this field?

Reply:
This is needed in https://github.com/build-trust/ockam/pull/3306. I've separated the `multiaddr` related changes to keep the other PR simpler.